ext/date/tests/bug74639.phpt

```diff
@@ -0,0 +1,33 @@
+--TEST--
+Bug #74639 Cloning DatePeriod leads to segfault
+--FILE--
+<?php
+
+$start = new DateTime('2017-05-22 09:00:00', new \DateTimeZone('Europe/Berlin'));
+$end = new DateTime('2017-08-24 18:00:00', new \DateTimeZone('Europe/Berlin'));
```

Comment:
@derickr thanks! to be fair we don't need to set timezone in this test at all. So I removed it, thanks